site/topics/testing/testing.rst

```diff
@@ -25,6 +25,55 @@ Testing
 Writing Tests
 -------------
 
+* For simplicity, we will keep our testing strategy to assertions about the data
+* Let's say we want to test the absolute value function ``abs``
+
+    * This function is provided to you by Python, so there is no actual need to test it here
+
+* If we are to consider how we could test the ``abs`` function, we want to check that the function output (returns) matches what it should be based on the input (parameters)
```

Comment:
So common thing people get wrong here, _parameters_ are part of the function's interface, _arguments_ are the input to a function invocation. `f(x)`, `x` is the parameter, `f(2)`, `2` is the argument. You're doing case testing here, so it should probably be arguments here. There exists better strategies for testing where you actually test the interface of a function, but they're well beyond the scope of this course.

Reply:
Legit didn't know there was an actual difference.

Reply:
Wow, we never stop learning new things.